.github/workflows/develop_commit_notification.yml

```diff
@@ -0,0 +1,15 @@
+name: Send info about commit to develop
```

Comment:
Could you explain more what this does? I don't understand it

Reply:
I've rephrased this. Basically, this job notifies the release-scripts repo and then the release-scripts repo will start building the release candidate.